capstone/cap/models.py

```diff
@@ -251,22 +251,36 @@ class CaseMetadata(models.Model):
     slug = models.SlugField(unique=True, max_length=255)
     first_page = models.IntegerField(null=True, blank=True)
     last_page = models.IntegerField(null=True, blank=True)
-    jurisdiction = models.ForeignKey('Jurisdiction', null=True, related_name='%(class)s_jurisdiction', on_delete=models.SET_NULL)
-    citation = models.CharField(max_length=255, blank=True)
+    jurisdiction_xml = models.CharField(max_length=255, blank=True)
```

Comment:
Oh, I see. That makes sense. I don't think we should label it XML unless there's XML in there. Maybe jursdiction_value?